cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -155,10 +177,13 @@ public static CruiseControl fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup ver
             cruiseControl.setTlsSidecar(tlsSidecar);
 
             cruiseControl = updateConfiguration(spec, cruiseControl);
-            KafkaConfiguration configuration = new KafkaConfiguration(kafkaSpec.getKafka().getConfig().entrySet());
+
+            KafkaClusterSpec kafkaClusterSpec = kafkaSpec.getKafka();
+            KafkaConfiguration configuration = new KafkaConfiguration(kafkaClusterSpec.getConfig().entrySet());
             if (configuration.getConfigOption(MIN_INSYNC_REPLICAS) != null) {
                 cruiseControl.minInsyncReplicas = configuration.getConfigOption(MIN_INSYNC_REPLICAS);
             }
+            cruiseControl.diskCapacityPerBroker = estimateDiskCapacityPerBrokerInMb(kafkaClusterSpec.getStorage());
```

Comment:
is this related to liveness and readiness probes? If not, why it's in this PR? I would open a different one.